code/__DEFINES/flags.dm

```diff
@@ -23,6 +23,10 @@
 
 #define NOSLIP		1024 		//prevents from slipping on wet floors, in space etc (NOTE: flag shared with THICKMATERIAL for external suits and helmet)
 
+#define OPENCONTAINER		4096 //is an open container for chemistry purposes
+
+#define INJECTONLY		8192

```

Comment:
No, do not make this a flag. Make it a secondary flag or a variable that works with opencontainer please.